components/button/button.css

```diff
@@ -0,0 +1,100 @@
+.btn {
+  margin: 0.5rem;
```

Comment:
Dont give margin and padding both inside component. Try to handle margin between elements using flex gap to the outer container